pkg/api/v1/schema.go

```diff
@@ -228,6 +228,8 @@ type Devices struct {
 	Watchdog *Watchdog `json:"watchdog,omitempty"`
 	// Interfaces describe network interfaces which are added to the vmi.
 	Interfaces []Interface `json:"interfaces,omitempty"`
+	// Tablet describe tablet device
+	Tablet *Tablet `json:"tablet,omitempty"`
```

Comment:
I'm unsure about how to best expose this option via our API.

Should we consider an array of input devices?  That's what we do for disks and interfaces.

Reply:
Probably. We likely won’t use many and settle for virtio-input (or alike) for linux and win with drivers and usb+tablet for bare win and “other” OSes. But I guess array makes sense. What I would suggest though is to name it as “input” rather than “tablet”. Or something else denoting “an input device” as you see fit

Reply:
so, would something like this make sense for the api?

```
type Devices struct{
  . 
  .
  .
  Inputs []Input `json:"inputs,omitempty"`
  .
  .
  .
}

type Input struct {
  Type string
  Bus string
}
```

Reply:
My first idea was to make it generic for more types of input devices, because libvirt support more input devices https://libvirt.org/formatdomain.html#elementsInput. But then after discussion with @MarSik, we decided to support only tablet. But sure i can update it to support more types of input devices.